docs/resources/rdb_instance.md

```diff
@@ -174,7 +193,7 @@ Please consult the [GoDoc](https://pkg.go.dev/github.com/scaleway/scaleway-sdk-g
 
 ~> **Important:** Updates to `private_network` will recreate the attachment Instance.
 
-- `ip_net` - (Required) The IP network where to con.
+- `ip_net` - (Optional) The IP network where to connect. Service IP is handle is not set.
```

Comment:
Maybe review the doc format and the wording.
IPAM usage should be clear

Reply:
Thanks, its updated.